src/discretize.jl

```diff
@@ -401,7 +401,7 @@ to the PDE.
 For more information, see `discretize` and `PINNRepresentation`.
 """
 function SciMLBase.symbolic_discretize(pde_system::PDESystem,
-                                       discretization::PhysicsInformedNN)
+    discretization::PhysicsInformedNN; bayesian::Bool = false,dataset_given=[nothing])
```

Comment:
The bayesian flag doesn't feel right, it should be a new discretization type and have a separate dispatch for this function

Reply:
I was thinking that too, but the top level interface has to be different since it's not building an optimization problem. So this is mostly internal, so I think it's okay for simplification.